netlogo-core/src/main/api/LabProtocol.scala

```diff
@@ -23,14 +24,29 @@ case class LabProtocol(name: String,
 
   def elements: SettingsIterator = {
     def combinations(sets: List[ValueSet]): SettingsIterator =
-      sets match {
-        case Nil => Iterator(Nil)
-        case set::sets =>
-          set.iterator.flatMap(v =>
-            combinations(sets).map(m =>
-              (set.variableName,v) :: m))
-      }
-    combinations(valueSets)
+      useDefaultRunOrder match {
+      case true => 
+        sets match {
+          case Nil => Iterator(Nil)
+          case set::sets =>
+            set.iterator.flatMap(v =>
+              combinations(sets).map(m =>
+                (set.variableName,v) :: m))
+        }
+      case _ => 
+        sets match {
+          case Nil => Iterator(Nil)
+          case set::sets =>
+            set.iterator.flatMap(v =>
+              combinations(sets).map(m =>
+                  m :+ new Tuple2(set.variableName,v)))   
+        }
+    } 
+    if(useDefaultRunOrder) combinations(valueSets)
       .flatMap(Iterator.fill(repetitions)(_))
+    else {
+      val runners = combinations(valueSets.reverse).toList
+      (for(i <- 1 to repetitions) yield runners).flatten.toIterator
+    }
```

Comment:
This function branches on `useDefaultRunOrder` once above (in the inner function) and once here. Perhaps it should be two functions instead: `sequentialCombinations` and `alternatingCombinations`?